tensorflow/matmul.py

```diff
@@ -141,7 +149,8 @@ def main():
             "rate": rate,
             "elapsed": elapsed,
         },
-        "tensorflow_buildinfo": _get_aicoe_tensorflow_build_info()
+        "tensorflow_buildinfo": _get_aicoe_tensorflow_build_info(),
```

Comment:
This to `tensorflow_aicoe_buildinfo` right?

Reply:
yes :+1: